src/server/utils/missing-countries.ts

```diff
@@ -0,0 +1,138 @@
+import { prisma } from "../models/db/prisma-client";
+import { logger } from "../services/logger";
+import { createList } from "../models/list";
+import { countriesList } from "../services/metadata";
+import { isTest } from "../config";
+
+import { ServiceType } from "../models/types";
+import type { CountryName, List } from "../models/types";
+
+const DEFAULT_USER_EMAIL = "ali@cautionyourblast.com";
+
+export const errorMessages = {
+  serviceType: "Incorrect service type entered",
+  newCountries: "No new countries are needed for this service type",
+} as const;
+
+/**
+ * This script compares the number of countries a service has in the db with what's in the countriesList metadat.ts file
+ * and adds countries that are missing to the db for that specific service.
+ *
+ * @example npm run add-missing-countries -- --service lawyers
+ * @example npm run add-missing-countries -- --service lawyers --emails "ali@cautionyourblast.com, jen@cautionyourblast.com"
+ */
+async function addMissingCountriesScript(): Promise<void> {
+  let emails: string[] | undefined;
+  const argumentOneIdentifier = process.argv[3];
+  const argumentOneValue = process.argv[4];
+  const argumentTwoIdentifier = process.argv[5];
+  const argumentTwoValue = process.argv[6];
+
+  if (argumentOneIdentifier !== "--service") {
+    logger.error(
+      'Service type not correctly provided, try adding "-- --service" to the command'
+    );
+    process.exit();
+  }
+
+  if (argumentTwoIdentifier === "--emails") {
+    const hasMultipleEmails = argumentTwoValue.includes(",");
+
+    emails = hasMultipleEmails
+      ? argumentTwoValue.split(",")
+      : [argumentTwoValue];
+  }
+
+  const serviceType = argumentOneValue;
+
+  await addMissingCountriesToService(serviceType, emails);
```

Comment:
can you add an extra step to get the emails from an existing provider list so you can copy into the new list.  For example, add another argument, copyFromList that will get the email addresses from that list and copy them over to the new list.  

`serviceType` <mandatory>: translatorsInterpreters
`copyFrom` <optional>: lawyers
`emails`: <optional>: we won't specify this when running it for our use-case

when the process runs it will do as it does today but also extract the permissions set for the same country in the lawyers list and copy to the translatorsInterpreters list.

Hope that makes sense?

Reply:
Ah this is what you said originally in the ticket, I didn't quite get it at first but now it makes more sense, (I guess I understand the project a bit better now 😁 )

Reply:
Actually... there might be loads of lists for a specific service. 

<img width="1808" alt="Screenshot 2022-08-22 at 11 40 39" src="https://user-images.githubusercontent.com/1377253/185902503-685bf9ba-3f37-46f9-a023-f512d9a55939.png">

Would it make sense to pick the first list for the service type? If not we could add country as an optional parameter, so like
`"--copyFrom" "lawyers, France"`

But that might be too much work 🤔